src/products.js

```diff
@@ -0,0 +1,59 @@
+import sandwiches from './data/sandwiches.js';
+import renderSandwiches from './render-sandwich.js';
+
+const showAllCategory = document.getElementById('show-all-category');
+const hotCategory = document.getElementById('hot-category');
+const coldCategory = document.getElementById('cold-category');
+const breakfastCategory = document.getElementById('breakfast-category');
+const eachSandwich = document.getElementById('product-list');
+let listItem;
+let i;
+
+// Loop to display all sandwich options
+for(let i = 0; i < sandwiches.length; i++) {
+    const sandwich = sandwiches[i];
+    listItem = renderSandwiches(sandwich);
+    eachSandwich.appendChild(listItem);
+}
+
+// Buttons to sort categories
+showAllCategory.addEventListener('click', () => {
+    const allSandwiches = eachSandwich.querySelectorAll('li');
```

Comment:
Great job on the bonus! A simple way to eliminate some repetition would be to declare allSandwiches above your event listeners so that you only have to declare it once (it will be in the scope of all of the event listeners if you declare it first).